pyproject.toml

```diff
@@ -143,6 +143,12 @@ select = [
 "__init__.py" = [
     "F401",
 ]
+"*.pyi" = [
+    "PYI011",
```

Comment:
Wow, why are ignoring all these lints?

Reply:
If these weren't picking on genuine errors because of typing imports we should report this to the ruff team.